src/dataflow/render/mod.rs

```diff
@@ -275,10 +274,12 @@ pub(crate) fn build_dataflow<A: Allocate>(
                         }
                     };
 
+                    let err_collection = Collection::empty(region);
```

Comment:
Maybe flag this with a comment or the like about how this is what needs to get swapped out should we want to capture errors from an input source?

Reply:
Good point. Done.